### PR TITLE
fix: contentBase (static) logging

### PR DESCRIPTION
**Pull request by ylemkimon**
- [x] This is a **bugfix**
- [ ] This is a **feature**
- [ ] This is a **code refactor**
- [ ] This is a **test update**
- [ ] This is a **docs update**
- [ ] This is a **metadata update**

### For Bugs and Features; did you add new tests?
N/A (covered by existing tests)

### Motivation / Use-Case
`contentBase` has been removed in favor of `static` in #2670.

### Breaking Changes
N/A

### CODE REVIEW

**Review by evilebottnawi — APPROVED** [review]
/cc @hiroppy @Loonride

**Review by hiroppy — APPROVED** [review]

**Review by rishabh3112 — APPROVED** [review]

File: lib/utils/status.js [hunk 1]
```diff
@@ -21,11 +17,11 @@ function status(uri, options, logger, useColor) {
     `webpack output is served from ${colors.info(useColor, options.publicPath)}`
   );
 
-  if (contentBase) {
+  if (options.static && options.static.length) {
```
Comment (evilebottnawi) [inline lib/utils/status.js:20]: Let's improve this `options.static.length > 0`

Comment (ylemkimon) [reply lib/utils/status.js:20]:
```suggestion
  if (options.static && options.static.length > 0) {
```

**Review by evilebottnawi — APPROVED** [review]

**Comment by evilebottnawi** [comment]
Thanks

**Comment by evilebottnawi** [comment]
Something else where you want to help?

**Comment by ylemkimon** [comment]
@evilebottnawi Yes, please let me know where I can help.

**Comment by evilebottnawi** [comment]
@ylemkimon Two PRs:
- https://github.com/webpack/webpack-dev-server/pull/2761
- https://github.com/webpack/webpack-dev-server/pull/2811 (tests failed)

**Pull request merged**